bin/gpodder

```diff
@@ -39,7 +39,6 @@ logger = logging.getLogger(__name__)
 
 try:
     import dbus
-    from dbus.mainloop.glib import DBusGMainLoop
```

Comment:
that one may need to stay for full dbus detection, don't you think?

Reply:
I am using the fake dbus module instead of python-dbus and `have_dbus` is True with or without the DBusGMainLoop import.

Reply:
I'm somewhat out of my depth here, not having really used dbus-python, but is there a situation where `dbus` could be imported but `dbus.mainloop.glib` not? If this line is really needed, then it can be marked with `noqa: F401` comment.

Reply:
ok, I added DBusGMainLoop import in e4a16f152 but as far as I can tell [from the code](https://github.com/freedesktop/dbus-python/blob/dbus-python-1.2.18/dbus_glib_bindings/module.c#L180) it should be instantiated to do anything useful.